x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/BasicSearchableSnapshotsIntegTests.java

```diff
@@ -53,39 +49,7 @@
 import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.Matchers.lessThanOrEqualTo;
 
-public class SearchableSnapshotsIntegTests extends ESIntegTestCase {
```

Comment:
I prefer that we keep the previous name `SearchableSnapshotsIntegTestCase` and to have a prefixed abstract class, like we usually do

Reply:
Thanks, yes, that's better. Done in eb73041.